# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/workers/test/crashtests/crashtests.list
# Commit: 588d5d470544
# Full Hash: 588d5d4705444710c4a6c35290fc92d802557776
# Author: Olli Pettay <Olli.Pettay@helsinki.fi>
# Date: 2023-03-11 09:54:30
# Description:
#   Bug 1819146, don't try to use destroyed global when accessing signal's reason, r=peterv
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D172131
# ==============================================================================

diff -r 6ac0d009f1ef -r 588d5d470544 dom/workers/test/crashtests/crashtests.list
--- a/dom/workers/test/crashtests/crashtests.list	Fri Mar 10 15:00:39 2023 +0000
+++ b/dom/workers/test/crashtests/crashtests.list	Fri Mar 10 15:14:32 2023 +0000
@@ -4,3 +4,4 @@
 load 1158031.html
 load 1228456.html
 load 1348882.html
+load 1819146.html
